### ISSUE TITLE
Feature request: .jpg ingest

### ISSUE THREAD

**Issue by sjohnson-imi** [usc-imi/aeo-light#24]
Request to the authors of the project to include support for JPG files.
When scanning in TIFF we get very large volumes.

_Originally posted by @dan7455 in https://github.com/usc-imi/aeo-light/issues/22#issuecomment-503507727_

**Comment by dan7455** [comment]
Will wait for

**Comment by dan7455** [comment]
Are there any advances in this issue?

**Comment by dan7455** [comment]
Hi. Are there any advances in this issue?

**Comment by dan7455** [comment]
Hi. Are there any advances in this issue?

**Comment by wilsbacher** [comment]
We are contemplating a wholistic revision to the input structure that would accommodate .jpg. In the meantime, of course, TIF and DPX are supported (in addition to video input).

**Comment by Ana77777** [comment]
I join the author of the request!

**Comment by meantux** [comment]
The worst part of this issue for me is the fact that Blackmagick's DaVinci Resolve stopped supporting the export to DPX files (the older Fusion 9 was doing it), unless I'm just lost in their newer interface). 
Now I have the supplemental step of converting the JPGs to DPX in my workflow. (it's easy with ImageMagick's convert tool but it stills consume time and effort).
Supporting the JPGs would be a major improvement for me.

**Comment by RussellC23** [comment]
I’ve been getting great results with png sequences.

RC

**Comment by retokromer** [comment]
> the fact that Blackmagick's DaVinci Resolve stopped supporting the export to DPX files

This is not correct. Of course, export to various DPX flavours still works fine in DaVinci Resolve 16.1.2.

**Comment by Ana77777** [comment]
When will the update with the choice of jpg

**Comment by JeromeMartinez** [comment]
> When will the update with (put here any request)

Classic repeated question, generic answer: when someone is really interested and either develop or sponsor a developer for speeding up the development the requested feature, if this is not the priority for someone else e.g. the main developers (because they don't need the feature), I guess.
The ball is in your court.
